site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -189,6 +189,10 @@ for more.
 
 > Warning: This section is TBD. PRs and discussions welcome.
 
+See the [`popover=hint` explainer's accessibility section](../popover-hint.research.explainer/#a11y-api-support) for additional comments. In particular, many of the relationships that are provided for "regular" invokers such as `popovertarget` apply to interest-based invokers also. For example, an `aria-details` relationship should be established between an interest invoker and its target.
+
+In particular, for interest-based triggering of popovers, there is a problem with hidden objects. In general, the details relation does not work for hidden objects, and we can’t necessarily expect the popover to be complete before it becomes visible, as that may happen on-demand. So what happens if the user navigates to a popover’s trigger, and it’s not open. If they tabbed there, then we can make sure the popover is shown on focus. But, if they are navigating with a virtual buffer as is likely, how will they trigger the hint popover to open (the user would need to know to focus the trigger, or JAWS/NVDA would need to start focusing objects during navigation as other screen readers do), and then and how will the user know it’s there (should we use a new value of the haspopup object attribute for these, e.g. `haspopup=hint`)?
```

Comment:
Filed https://issues.chromium.org/issues/377923492

Reply:
Thanks! I wasn't sure if you wanted this included, so I went ahead and included it.